packages/supplementary-contracts/script/blacklist/Deploy.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+import "forge-std/src/Script.sol";
+import "forge-std/src/console2.sol";
+
```

Comment:
After deployment please make sure you update this file with the code's commit hash and the transaction's hash - https://github.com/taikoxyz/taiko-mono/blob/main/packages/protocol/deployments/mainnet-contract-logs.md